packages/swap/src/routes/v2/utils.ts

```diff
@@ -176,8 +176,15 @@ export const getDepositInfo = (
       witnessedAt:
         swapRequest?.originType === 'VAULT'
```

Comment:
same here, we can probably just remove this condition, especially considering the current behavior of swaps being requested in the same block as they are witnessed

Reply:
good with be too - updated